src/otx/algo/detection/atss.py

```diff
@@ -185,13 +159,8 @@ def _customize_outputs(
 
     def get_classification_layers(self, prefix: str = "model.") -> dict[str, dict[str, int]]:
         """Get final classification layer information for incremental learning case."""
-        from otx.core.utils.build import modify_num_classes
-
-        sample_config = deepcopy(self.config)
-        modify_num_classes(sample_config, 5)
-        sample_model_dict = TorchATSS(**convert_conf_to_mmconfig_dict(sample_config)).state_dict()
-        modify_num_classes(sample_config, 6)
-        incremental_model_dict = TorchATSS(**convert_conf_to_mmconfig_dict(sample_config)).state_dict()
+        sample_model_dict = self._build_model(num_classes=5).state_dict()
+        incremental_model_dict = self._build_model(num_classes=6).state_dict()
```

Comment:
Sorry for missing the background behind these numbers `num_classes=5 or 6`. Could you explain?

Reply:
The overall function is for finding final classification layer names. If the layer's state dict shape is changed due to num_classes changes, that means it is one of the final classification layer.

Reply:
@harimkang, could you consider a way for better loading of pretrained weight?